tests/run_tests.sh

```diff
@@ -10,4 +10,4 @@
 # not expressly granted therein are reserved by Shotgun Software Inc.
 
 find . -name "*.pyc" -delete
-python  `dirname $0`/run_tests.py $*
+python -3 `dirname $0`/run_tests.py $*
```

Comment:
i cannot comment on the `tests/run_tests25.sh` file so i say it here instead: YESSSSSSSSSSS!